addons/mail/static/src/messaging/test_utils.js

```diff
@@ -530,7 +530,7 @@ async function pause() {
  * @param {Object} [param0.viewOptions] makes only sense when `param0.hasView`
  *   is set: the view options to use in createView.
  * @param {boolean} [param0.waitUntilMessagingInitialized=true]
- * @param {boolean} [param0.withMailbotHasRequestDefaultPatch]
+ * @param {Object} [param0.withWindowNotification]
```

Comment:
I think this param could be `window.Notification`, which passes a full mocked window.Notification object.
Example of use:
```js
await start({
   'window.Notification': {
      permission: 'default',
      async requestPermission() { this.permission = 'denied'; },
   },
});
```

if not provided, uses either genuine `window.Notification` or uses a default mocked version.

Reply:
refresh and check again